src/NBomber/Domain/Errors.fs

```diff
@@ -128,6 +130,12 @@ type AppError =
 
         | EnterpriseOnlyFeature message -> message
 
+        | EmptyThresholds scenarioName ->
+            $"Scenario: '{scenarioName}' has no thresholds"
+
+        | DuplicateThresholdName(scenarioName, thresholdName) ->
+            $"Scenario: '{scenarioName}' contains thresholds with duplicated name: '{thresholdName}'"
```

Comment:
![image](https://user-images.githubusercontent.com/17567634/169595963-bfe27e4e-9e5b-4fa5-a384-191b5cd07253.png)

Reply:
This is not relevant anymore as I'm going to remove this particular validation.